wavefrontEmitter.py

```diff
@@ -240,39 +243,54 @@ def parse_collector(self, message):
         message - a JSON object representing the message sent to datadoghq
         """
 
-        tstamp = long(message['collection_timestamp'])
-        host_name = message['internalHostname']
+	if 'collection_timestamp' in message:
+            tstamp = long(message['collection_timestamp'])
+        elif 'timestamp' in message:
+	    tstamp = long(message['timestamp'])
+        if 'internalHostname' in message:
+            host_name = message['internalHostname']
+        elif 'host_name' in message:
+	    host_name = message['host_name']
 
         # cpu* mem*
-        for key, value in message.iteritems():
-            if key[0:3] == 'cpu' or key[0:3] == 'mem':
-                dotted = 'system.' + emitter.convert_key_to_dotted_name(key)
-                self.send_metric(dotted, value, tstamp, host_name, None)
+        if isinstance(message[0],dict):
```

Comment:
if isinstance(message,list):
   if isinstance(message[0],dict):
       for key, value in message[0].iteritems():